pkg/endpoints/websocket_test.go

```diff
@@ -490,4 +480,48 @@ func CUDExtensions(r *Resource, t *testing.T, namespace string) {
 	}
 }
 
+func CUDSecrets(r *Resource, t *testing.T, namespace string) {
```

Comment:
Can you make it a clear what CUD is through the code please? I'm guessing `controller updated secrets`. Also, this is a test but you've not included test anywhere in the function name so at a first glance anyone could easily think it's just a utility method

Reply:
Since there can potentially be many of these CUD/CD methods, I documented it inline where they should all be called rather than on each. I could modify this.